python/controllers/ncj_controller.py

```diff
@@ -26,3 +26,10 @@ def createNcjPool(request: JsonRpcRequest, template, parameters):
     print("Poolparm from json is done")
     client.pool.add(pool)
     return {"what": "it works"}
+
+@app.procedure("get-ncj-pool")
+def getNcjPool(request: JsonRpcRequest, template, parameters):
```

Comment:
can you call that `expand_ncj_pool` instead(and change at id also to be `expand-ncj-pool`